src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -34,7 +36,8 @@ const RadioButtonGroup = (props) => {
         checked,
         inputName: groupName,
         onChange: handleChange,
-        tabindex
+        tabindex,
+        groupName
```

Comment:
We don't need to pass down the `groupName` prop, its already passed as `inputName`

Reply:
Oh... how could I missed that one